Sources/MapboxCoreNavigation/EHorizon/EHorizonEdgeMetadata.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import MapboxNavigationNative
+
+public struct EHorizonEdgeMetadata {
+
+    /** The bearing in degrees clockwise at the start of the edge. */
+    public let heading: Double
```

Comment:
`MapboxDirections` uses `CLLocationDirection` for `headings`

Reply:
`CLLocationDirection` is measured starting from 0° at due north. `Measurement<UnitAngle>` is used for relative angles. What is the reference point for this heading?

/cc @SiarheiFedartsou

Reply:
It is the same as for other headings in our codebase - 0° at due north, i.e. `CLLocationDirection ` can be used here.